src/components/Header/Header.css

```diff
@@ -1,7 +1,32 @@
 .header {
     position: fixed;
     width: 100vw;
```

Comment:
while it's not much of an issue here because it's a small project, let's convert `.css` files into `.module.css` so that the styles only apply to the related `.jsx` file. this becomes an issue when you're importing styled components into other pages with their own styling, so names can be mixed up. reference this [article](https://medium.com/@ralph1786/using-css-modules-in-react-app-c2079eadbb87) for more + how to convert it!